src/plugins/intel_gpu/src/runtime/memory.cpp

```diff
@@ -38,6 +38,12 @@ memory::~memory() {
     }
 }
 
+event::ptr simple_attached_memory::copy_to(stream& /* stream */, void* host_ptr, bool /* blocking */) {
+    host_ptr = malloc(size());
+    memcpy(host_ptr, _pointer, size());
+    return nullptr;
```

Comment:
It's better to return user event instead of nullptr, we don't have checks for nullptr in all places for events